README.md

```diff
@@ -46,6 +44,29 @@ However, there are challenges with assembling and managing these tools yourself:
 
 The VMClarity project is focused on unifying detection and management of VM security threats in an agentless manner.
 
+# Quick start
```

Comment:
I want to leave this comment higher up, but github won't let me. There is a spelling mistake in the existing README text in the first sentence of "Why VMClarity":

"hypescalers" should be "hyperscalers"